src/mcp_server_obsidian_omnisearch/__main__.py

```
@@ -1,5 +1,23 @@
-from mcp_server_obsidian_omnisearch import main
 
-main()
+def main():
+    import sys
+    import asyncio
+
+    if len(sys.argv) < 2:
+        print("Usage: python server.py <obsidian_vault_path>")
+        sys.exit(1)
+
+    obsidian_vault_path = sys.argv[1]
```

Ruff (0.14.8)

Local variable `obsidian_vault_path` is assigned to but never used

Remove assignment to unused variable `obsidian_vault_path`

(F841)